scripts/update-nordix-repos-master.sh

```diff
@@ -153,16 +153,16 @@ for repo in ${UPDATE_REPO}; do
 done
 
 # Sync non default branches on selected repos
-CAPM3_RELEASE_BRANCHES="release-1.3 release-1.4 release-1.5 release-1.6"
+CAPM3_RELEASE_BRANCHES="release-1.3 release-1.4 release-1.5 release-1.6 release-1.7"
```

Comment:
```suggestion
CAPM3_RELEASE_BRANCHES="release-1.5 release-1.6 release-1.7"
```
Remove unsupported versions while at it, same for all vars.